__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is this empty line necessary?

Reply:
The __init__.py files are all empty by default. I found that when I removed any of them I get import errors in the test file and I looked it up and read that they are necessary in order to import files between folders.